tsconfig.json

```diff
@@ -15,6 +15,9 @@
   "include": [
     "src"
   ],
+  "exclude": [
+    ".vscode-test"
```

Comment:
What's this?

Reply:
that's unnecessary, I'll remove that, it's included in node_modules but that is already in .gitignore